extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -605,6 +612,57 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 
   /**
+   * Checks if request for a trigger is enabled.
+   * @param {!JSONType} trigger The config to use to determine sampling.
+   * @param {!Object} event Object with details about the event.
+   * @return {!Promise<boolean>} Whether trigger must be called.
+   * @private
+   */
+  checkTriggerEnabled_(trigger, event) {
+    const expansionOptions = this.expansionOptions_(event, trigger);
+    const enabledOnTagLevel =
+        this.checkSpecEnabled_(this.config_['enabled'], expansionOptions);
+    const enabledOnTriggerLevel =
+        this.checkSpecEnabled_(trigger['enabled'], expansionOptions);
+
+    return Promise.all([enabledOnTagLevel, enabledOnTriggerLevel])
+        .then(enabled => {
+          dev().assert(enabled.length === 2);
+          return enabled[0] && enabled[1];
+        });
+  }
+
+  /**
+   * Checks result of 'enabled' spec evaluation. Returns false if spec is provided and value
+   * resolves to empty string.
```

Comment:
I think we should go ahead and change this, yes. The only more desirable alternative is to change var/url-replacements expansion with a new mode that will take false-y values and instead of outputting them via `String(x)` would instead output them as an empty string. However, that'd be a lot of changes for a relatively small feature. So, in the meantime, using falsy strings is a good alternative, I believe.